contracts/actions/aaveV3/AaveV3SwapBorrowRateMode.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import "../../utils/TokenUtils.sol";
+import "../ActionBase.sol";
+import "./helpers/AaveV3Helper.sol";
+
+/// @title Swaps proxy positions borrow rate mode between stable and variable.
+contract AaveV3SwapBorrowRateMode is ActionBase, AaveV3Helper {
+    using TokenUtils for address;
+    struct Params {
+        address asset;
+        uint256 rateMode;
+        bool useDefaultMarket;
```

Comment:
this uses a slot, maybe have address(0) mean default market? the decode function could infer this from encodedInput.length